contrib/win32/win32compat/win32_usertoken_utils.c

```diff
@@ -382,4 +392,262 @@ int load_user_profile(HANDLE user_token, char* user) {
 	return r;
 }
 
+
+/* *** virtual account token generation logic ***/
+
+/* 
+ * these definitions are supposed to be part of Ntsecapi.h 
+ * but are missing due to a bug acknowledged by Windows team 
+ * These can be deleted once the SDK is fixed in future
+ */
```

Comment:
can we move them to a different file (definitions missing in Ntsecapi.h) so that it will be more readable and easy to delete the whole file later?

Reply:
makes sense. I'll do that.